project/ScalafixBuild.scala

```diff
@@ -127,35 +124,32 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     updateOptions := updateOptions.value.withCachedResolution(true),
     ThisBuild / watchTriggeredMessage := Watch.clearScreenOnTrigger,
     commands += Command.command("save-expect") { s =>
-      "unit/test:runMain scalafix.tests.util.SaveExpect" ::
+      "unitTarget2_13/test:runMain scalafix.tests.util.SaveExpect" ::
         s
     },
     commands += Command.command("ci-3") { s =>
-      s"""set testsInput/scalaVersion := "$scala3"""" ::
-        s"""set testsOutput/scalaVersion := "$scala3"""" ::
-        "unit/testOnly scalafix.tests.rule.RuleSuite" :: s
+      "unitTarget3/test" :: s
     },
     commands += Command.command("ci-213") { s =>
-      s"""set ThisBuild/scalaVersion := "$scala213"""" ::
-        "unit/test" ::
-        "docs/run" ::
-        "interfaces/doc" ::
+      "unitTarget2_13/test" ::
+        "unitTarget2_13/it:test" ::
+        "docs2_13/run" ::
+        "interfaces2_13/doc" ::
         testRulesAgainstPreviousScalaVersions(scala213, s)
     },
     commands += Command.command("ci-212") { s =>
-      s"""set ThisBuild/scalaVersion := "$scala212"""" ::
-        "unit/test" ::
+      "unitTarget2_12/test" ::
+        "unitTarget2_12/it:test" ::
         testRulesAgainstPreviousScalaVersions(scala212, s)
     },
     commands += Command.command("ci-211") { s =>
-      s"""set ThisBuild/scalaVersion := "$scala211"""" ::
-        "unit/test" ::
+      "unitTarget2_11/test" ::
+        "unitTarget2_11/it:test" ::
         testRulesAgainstPreviousScalaVersions(scala211, s)
     },
     commands += Command.command("ci-213-windows") { s =>
-      s"++$scala213" ::
-        "cli/crossPublishLocalBinTransitive" :: // scalafix.tests.interfaces.ScalafixSuite
```

Comment:
`crossPublishLocalBinTransitive` was handy in that particular case where we use `testOnly` which does not trigger `crossPublishLocalBinTransitive` (since it would be annoying locally). I thought we could extract this in the IT config (since it requires external stuff), which allows us to stop triggering that test via `testOnly` but through `it:test` which has now the proper hook.